projects/packages/jetpack-mu-wpcom/src/features/wpcom-wpadmin-page-view/wpcom-wpadmin-page-view.php

```diff
@@ -0,0 +1,188 @@
+		// Calypsoified WP Admin pages should not count as WP Admin page views, since the user is actually in Calypso.
+		if ( $is_calypsoify_enabled ) {
+			return;
+		}
+
+		$blog_id = $current_blog->blog_id;
+		// @phan-suppress-next-line PhanUndeclaredClassMethod -- This class is available in simple sites.
+		$user_types = \WPCOM_User::get_types();
+	}
+
+	if ( $is_atomic_site ) {
+		$blog_id    = _wpcom_get_current_blog_id();
+		$user_types = wpcom_atomic_get_user_types();
+	}
+	?>
+	<script type="text/javascript">
+		var _admin_pv_props = {
+			from_page: '<?php echo esc_js( $current_screen->id ); ?>',
+			is_block_editor: '<?php echo esc_js( $current_screen->is_block_editor ? 'true' : 'false' ); ?>',
+			source: 'wp-admin',
+			blog_id: '<?php echo esc_js( (string) $blog_id ); ?>',
+			user_type: '<?php echo esc_js( implode( ',', $user_types ) ); ?>'
+		};
+		_tkq = window._tkq || [];
+		_tkq.push( [ 'identifyUser', <?php echo (int) $current_user->ID; ?>, '<?php echo esc_js( $current_user->user_login ); ?>' ] );
+		_tkq.push( [ 'recordEvent', 'wpcom_admin_page_view', _admin_pv_props ] );
+	</script>
+	<?php
+}
+add_action( 'admin_footer', __NAMESPACE__ . '\wpcom_nosara_track_admin_page_views' );
+
+/**
+ * Track non-calypso Customizer views if the user is linked from the frontend.
+ */
+function wpcom_maybe_track_customizer() {
+	// not when the Customizer is loaded in Calypso
+	if ( isset( $_GET['calypso'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		return;
+	}
+
+	// links from frontend
+	if ( isset( $_GET['url'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		return wpcom_track_customizer_from_frontend();
+	}
+
+	// standard wp-admin
+	wpcom_nosara_track_admin_page_views();
+}
+
+/**
+ * Since links to the Customizer from the frontend aren't properly wp-admin (no wp-admin chrome,
+ * you return to the frontend after closing), they shouldn't be counted as wp-admin views.
+ */
+function wpcom_track_customizer_from_frontend() {
+	global $current_user;
+
+	// Since we view so many screens to offer support and do feature dev, I think it's best not to count us.
+	if ( do_not_track_a11ns() ) {
+		return;
+	}
+	?>
+	<script type="text/javascript">
+		_tkq = window._tkq || [];
+		_tkq.push( [ 'identifyUser', <?php echo (int) $current_user->ID; ?>, '<?php echo esc_js( $current_user->user_login ); ?>' ] );
+		_tkq.push( [ 'recordEvent', 'wpcom_customize_loaded_from_frontend' ] );
+	</script>
+	<?php
+}
+add_action( 'customize_controls_print_footer_scripts', __NAMESPACE__ . '\wpcom_maybe_track_customizer' );
+
+/**
+ * Determine if the current user should not be tracked.
+ *
+ * @return bool
+ */
+function do_not_track_a11ns() {
+	$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
+	$is_atomic_site = ! $is_simple_site;
+
+	if ( $is_simple_site ) {
```

Comment:
We can simplify this to:

```
       $is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;

	if ( $is_simple_site ) {
                 return is_automattician() || is_network_admin();
	}
       
        return wpcom_atomic_maybe_is_a11n();
```

Besides simplifying the condition, if the user is not a11n or network admin, we'll return false instead of void.

We could even one-line the whole thing, but it's easier to read this way.